m4_sessions/1710-inning/group_five.md

```diff
@@ -1,10 +1,28 @@
 Group Five
 
-Session Title: 
+Session Title: Crushing Cold Outreach
+
 Description: Why should students come to this session? 
+Are you terrified of cold outreach? Or do you not even know what it is? We'll be doing a live walk through with volunteers of successfully reaching out to people you don't know. We'll address feelings of rejection, imposter syndrome, expectations and targeting strtategies. 
 
 Session Outcomes: What will students leave with or gain from this session?
+- everyone leaves with one message sent or drafted
+- good understanding of future approach
+- mitigation of anxiety around rejection
+
 
 Session Outline:  Include the flow of the session and the time for each section.
+1. Brief intro of what it is, why it useful
+2. fist to 5 about how comfortable you are with it
+3. discussion on why ppl feel comfortable or uncomfortable
+4. general strategies/steps/expectations
+5. live group demo
+6. 3 small groups - work on your own cold outreach
+7. large group - discuss issues that came up
```

Comment:
Def leave time for this.  Maybe there is even a discussion on "most common challenges" and then brainstorm ways to get through those.